cmd/loop/main.go

```diff
@@ -176,11 +176,22 @@ func parseAmt(text string) (btcutil.Amount, error) {
 }
 
 func logSwap(swap *looprpc.SwapStatus) {
-	fmt.Printf("%v %v %v %v - %v\n",
+	fmt.Printf("%v %v %v %v - %v",
 		time.Unix(0, swap.LastUpdateTime).Format(time.RFC3339),
 		swap.Type, swap.State, btcutil.Amount(swap.Amt),
 		swap.HtlcAddress,
 	)
+
+	if swap.State != looprpc.SwapState_INITIATED &&
```

Comment:
Also in case of a failure, there can be costs.

Incrementally showing may confuse users. When our off-chain is pulled, but the on-chain sweep hasn't confirmed, the server balance is high.